src/lang/en-US/Workflows.ts

```diff
@@ -77,9 +77,9 @@ export const Workflows: Record<string, string> = {
     'workflows.collectionSelector.manualBackfill.notSupported.title': `This {entityType} doesn’t support backfills.`,
     'workflows.collectionSelector.manualBackfill.notSupported.message': `To backfill, disable each binding, save and then re-enable and save.`,
     'workflows.collectionSelector.manualBackfill.message.capture': `Trigger a backfill of this collection from the source when published.`,
-    'workflows.collectionSelector.manualBackfill.message.capture.allBindings': `Trigger a backfill of all collections from the source when published.`,
+    'workflows.collectionSelector.manualBackfill.message.capture.allBindings': `Trigger a backfill of all collection from the source when published. The UI will mark all collections as backfilled but the server will filter out those that cannot be backfilled (e.g. disabled collections).`,
```

Comment:
The updated message is sufficient. Calling out that _collection_ is singular when it should be plural.